klasses/models.py

```diff
@@ -82,6 +85,34 @@ def payment_deadline(self):
         """
         return self.installment_set.aggregate(payment_deadline=models.Max('deadline'))['payment_deadline']
 
+    @property
+    def next_installment(self):
+        """
+        Get the next installment
+        """
+        return self.installment_set.exclude(
```

Comment:
This is a good point: I am not sure how we should consider installment with deadline=None: as open ended? And in this case should we just consider as deadline the Klass start date?